sys/include/net/gnrc/ipv6/ext/rh.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_ipv6_ext_rh Support for IPv6 routing header extension
+ * @ingroup     net_gnrc_ipv6_ext
+ * @brief       GNRC implementation of IPv6 routing header extension.
+ * @{
+ *
+ * @file
+ * @brief   GNRC routing extension header definitions.
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef NET_GNRC_IPV6_EXT_RH_H
+#define NET_GNRC_IPV6_EXT_RH_H
+
+#include "net/ipv6/hdr.h"
+
+#include "net/ipv6/ext/rh.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum {
+    /**
+     * @brief   The routing header was successfully processed and this node
+     *          is the destination
+     */
+    GNRC_IPV6_EXT_RH_OK = 0,
+    /**
+     * @brief   The routing header was successfully processed and the packet
+     *          was forwarded or should be forwarded by the central RH handler.
```

Comment:
Something that "was forwarded" could have this node as destionation right? Does the `GNRC_IPV6_EXT_RH_OK` have priority over this result?

Reply:
Can you rephrase this? I'm not sure I understand.

Reply:
Ah I get it now

Reply:
Maybe I'm interpreting the "was forwarded" here not correct. To me it sounds as if a frame was forwarded by another node to this node. Where at that point both the conditions from this documentation and the option currently renamed to `GNRC_IPV6_EXT_RH_AT_DST` match the frame.

Reply:
Because it was both forwarded to this node and at the destination.

Reply:
Maybe the documentation update clarifies it better.